front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/displayIntervals.ts

```diff
@@ -0,0 +1,37 @@
+import type { PowerRestrictionRange } from 'applications/operationalStudies/consts';
+import type { RangedValue } from 'common/api/osrdEditoastApi';
+
+export default function formatIntervals(
+  formattedPathElectrificationRanges: RangedValue[],
+  powerRestrictionRanges: PowerRestrictionRange[]
+) {
+  const formattedRanges: RangedValue[] = [];
+
+  formattedPathElectrificationRanges.forEach((electrificationRange) => {
+    const overlappingRanges = powerRestrictionRanges.filter(
+      (range) => electrificationRange.end > range.begin && electrificationRange.begin < range.end
+    );
+
+    if (overlappingRanges.length === 0) {
+      formattedRanges.push({
+        begin: electrificationRange.begin,
+        end: electrificationRange.end,
+        value: 'NO_POWER_RESTRICTION',
```

Comment:
Use the const which is defined in PowerRestrictionsSelector
```suggestion
        value: NO_POWER_RESTRICTION
```

Reply:
done

Reply:
I think you forgot this one !

Reply:
done, there was a dependency cycle, but it was fixed by the creation of a consts file